test/action_controller/json_api/linked_test.rb

```diff
@@ -39,70 +38,76 @@ def setup_post
 
           def render_resource_without_include
             setup_post
-            render jsonapi: @post
+            render json: @post
```

Comment:
I've been debating whether the following adequately test this:

https://github.com/rails-api/active_model_serializers/blob/bff317686c7277e97eb7eabc42078bb3c81b973b/test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb#L14-L17

https://github.com/rails-api/active_model_serializers/blob/bff317686c7277e97eb7eabc42078bb3c81b973b/test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb#L113-L126